.github/workflows/ci-for-default-ns.yaml

```diff
@@ -74,7 +74,12 @@ jobs:
           kubectl --namespace rook-ceph scale deploy/rook-ceph-osd-0 --replicas=0
           # we need to sleep so the osd will be marked down before purging the osd
           sleep 5
-          kubectl-rook_ceph rook purge-osd 0 --force
+          kubectl rook_ceph rook purge-osd 0 --force
```

Comment:
similar changes require in another ci file `ci-for-diff-ns`

Reply:
DOne